pallets/eth-bridge/src/requests/outgoing.rs

```diff
@@ -514,7 +518,7 @@ impl<T: Config> OutgoingAddToken<T> {
                 Token::Address(types::H160(token_address.0)),
                 Token::String(symbol.clone()),
                 Token::String(name.clone()),
-                Token::UintSized(decimals.into(), 8),
+                Token::Uint(decimals.into()),
```

Comment:
Token::Uint or TokenWrapper::UintSized ?

Reply:
It's ok, because signature V3 uses encode instead of encode_packed. We have tests for abi encoding and if it's passed everything should be fine